javatests/arcs/core/entity/HandleManagerTestBase.kt

```diff
@@ -92,12 +138,18 @@ open class HandleManagerTestBase {
         storageKey = singletonRefKey
     )
 
-    private val setRefKey = RamDiskStorageKey("set-ent")
-    private val setKey = ReferenceModeStorageKey(
+    private val collectionRefKey = RamDiskStorageKey("set-ent")
+    private val collectionKey = ReferenceModeStorageKey(
         backingKey = backingKey,
-        storageKey = setRefKey
+        storageKey = collectionRefKey
     )
 
+    private val hatCollectionRefKey = RamDiskStorageKey("set-ent")
+    private val hatCollectionKey = ReferenceModeStorageKey(
+        // TODO: Don't use the same backing key as for the other collection.
```

Comment:
Is something stopping us from using a different backing key right now?

Reply:
There was a bug I couldn't figure out, but your comment above fixed it!  🎉